streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilder.java

```diff
@@ -2035,6 +2035,11 @@ private void updateSubscribedTopics(final Set<String> topics, final String logPr
         return maybeDecorateInternalSourceTopics(sourceTopicNames);
     }
 
+    public boolean hasNamedTopologies() {
+        // TODO
```

Comment:
I am assuming that this will be done in a later PR

Reply:
Yep, that's covered by [Pt. 2](https://github.com/apache/kafka/pull/10683) -- the PR is available for review, but it's on top of this PR so it's probably going to be difficult to review until this one is merged